mlrun/datastore/datastore_profile.py

```diff
@@ -132,6 +132,21 @@ def attributes(self):
         return attributes
 
 
+class DatastoreProfileV3IO(DatastoreProfile):
+    type: str = pydantic.Field("v3io")
+    v3io_access_key: typing.Optional[str] = None
+    _private_attributes = "v3io_access_key"
+
+    def url(self, subpath):
+        return f"v3io://{subpath}"
+
+    def secrets(self) -> dict:
+        res = {}
+        if self.v3io_access_key:
+            res["V3IO_ACCESS_KEY"] = self.v3io_access_key
```

Comment:
What about `V3IO_API`?

Reply:
The V3IO_API is consistently included in the configuration. To manage various permissions for different profiles, we support storing multiple v3io keys. However, the v3io system, including its API, remains singular throughout MLRun.